.vale/styles/Vocab/Che/accept.txt

```diff
@@ -37,7 +37,6 @@ AsciiDoc
 autostart
 AWS|aws
 Bitbucket
-boolean
```

Comment:
Why do we remove this? For different purposes, Boolean and boolean are accepted.

Reply:
Indeed we need `[Bb]oolean` in the accept.txt to add it to the dictionary (`Spelling` rules) and accept both forms.

Reply:
Ah! Good! Sorry for missing it